gen/azure/cloud-config.yaml

```diff
@@ -19,6 +19,7 @@ root:
       Restart=always
       StartLimitInterval=0
       RestartSec=15
+      ExecStartPre=-/sbin/ip link del docker0
```

Comment:
See the linked `cloud_images/centos7/install_prereqs.sh` changes in this PR for the corresponding AWS EL7 AMI updates.